packages/svelte/elements.d.ts

```diff
@@ -2059,3 +2059,5 @@ export interface SvelteHTMLElements {
 
 	[name: string]: { [name: string]: any };
 }
+
+export type ClassValue = import('clsx').ClassArray | import('clsx').ClassDictionary;
```

Comment:
Since we are exporting this I think it would be easier to also include string here right? So whoever is using it can just use this without adding string by itself

Reply:
Good catch